File: 2114-maximum-number-of-words-found-in-sentences/2114-maximum-number-of-words-found-in-sentences.java
class Solution {
    public int mostWordsFound(String[] sentences) {
        int maxWordsNum = 0;
        for (String sentence : sentences) {
            String[] split = sentence.split(" ");
            int wordsNum = split.length;
            
            if (wordsNum > maxWordsNum)
                maxWordsNum = wordsNum;
        }
        return maxWordsNum;
    }
}

File: 2114-maximum-number-of-words-found-in-sentences/2114-maximum-number-of-words-found-in-sentences.cs
public class Solution {
    public int MostWordsFound(string[] sentences) {
        int maxWordsNum = 1;
        
        foreach (String sentence in sentences)
        {
            int wordsNum = 1;
            
            for (int i = 0; i < sentence.Length; i++)
            {
                if (sentence[i] == ' ')
                    wordsNum++;
            }
            maxWordsNum = wordsNum > maxWordsNum ? wordsNum : maxWordsNum;
        }
        return maxWordsNum;
    }
}